docs/src/rsh/consensus/index.md

```diff
@@ -628,9 +628,31 @@ only be constructed in a consensus step.
 
 A `{!rsh} Set` may be modified by writing `{!rsh} s.insert(ADDRESS)` to install `{!rsh} ADDRESS` in the
 set, `{!rsh} s`, or `{!rsh} s.remove(ADDRESS)` to remove the `{!rsh} ADDRESS` from the set.
+
+The following example shows the usage of `{!rsh} s.insert(ADDRESS)`:
+
+```reach
+load: /hs/t/y/merit-badge.rsh
```

Comment:
Do I recall correctly that you can't have a leading `/`? Or is that you need one? @TheChronicMonster